runtime/src/weights/pallet_utility.rs

```diff
@@ -32,22 +32,22 @@ use frame_support::{
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::weights::WeightInfo for WeightInfo<T> {
     fn batch(c: u32) -> Weight {
-        (8_254_000 as Weight)
-            // Standard Error: 30_000
-            .saturating_add((6_984_000 as Weight).saturating_mul(c as Weight))
+        (19_244_000 as Weight)
+            // Standard Error: 15_000
+            .saturating_add((7_387_000 as Weight).saturating_mul(c as Weight))
     }
     fn as_derivative() -> Weight {
-        (3_410_000 as Weight)
+        (4_040_000 as Weight)
     }
     // Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
     fn batch_all(c: u32) -> Weight {
-        (25_449_000 as Weight)
-            // Standard Error: 63_000
-            .saturating_add((7_641_000 as Weight).saturating_mul(c as Weight))
+        (0 as Weight)
```

Comment:
Is this as expected? I checked, and this is non-zero in the weights.rs of pallet-utility version 4.0.0-dev.

Reply:
This is out of my control, sometimes the benchmarks for the same benchmark logic start with a base weight of greater than zero and sometimes it's zero. I think as long as the weight is modified properly by variables this is acceptable. If a change in the logic did not happen we can revert the base weight tough, I think it's better to overcharge than to undercharge (undercharging can lead to overweight blocks that can't be produced in time).